tools/mmp/driver.cs

```diff
@@ -998,6 +998,8 @@ static int Compile (IEnumerable<string> internalSymbols)
 					args.Append ("-g ");
 				args.Append ("-mmacosx-version-min=").Append (minos.ToString ()).Append (' ');
 				args.Append ("-arch ").Append (arch).Append (' ');
+				if (arch == "x86_64")
```

Comment:
This is unrelated to the other registrar changes, so could this go into a separate commit/PR?